extensions/drupal-org/package.json

```
@@ -27,6 +27,13 @@
       "subtitle": "Drupal",
       "description": "Search Drupal API documentation",
       "mode": "view"
+    },
+    {
+      "name": "search-projects",
+      "title": "Search Projects",
+      "subtitle": "Drupal",
+      "description": "Search for Drupal projects",
```

Minor: I'd love it that someone can type in "module" or "theme" and be able to select this command. With that intent, what do you think of this change?

```suggestion
      "description": "Search for Drupal projects (modules, themes, etc)",
```

---

Yeah that sounds like a great idea!